lighthouse-core/audits/splash-screen.js

```diff
@@ -29,7 +29,9 @@ class SplashScreen extends MultiCheckAudit {
       name: 'splash-screen',
       description: 'Configured for a custom splash screen',
       failureDescription: 'Is not configured for a custom splash screen',
-      helpText: 'A default splash screen will be constructed for your app, but satisfying these requirements guarantee a high-quality [splash screen](https://developers.google.com/web/updates/2015/10/splashscreen) that transitions the user from tapping the home screen icon to your app\'s first paint',
+      helpText: 'Meet these requirements to ensure a high-quality experience when ' +
```

Comment:
maybe its a good time to rephrase this a bit, "Meet these requirements" diverges quite a bit from our usual style here.

wdyt about something like "A themed splash screen ensures a high-quality experience when..."?

Reply:
False

Reply:
ping @kaycebasques

Reply:
done